dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-2.0/resources/l10n/com.ibm.websphere.appserver.mpMetrics-2.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2017, 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides support for the Eclipse MicroProfile Metrics specification for enterprise Java.
```

Comment:
Description should should say it is version `2.0` and therefore update the copyright date